src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkRealConnectionMVCIT.java

```diff
@@ -0,0 +1,67 @@
+package com.researchspace.webapp.integrations.fieldmark;
+
+import static com.researchspace.service.IntegrationsHandler.FIELDMARK_APP_NAME;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.researchspace.api.v1.controller.API_MVC_TestBase;
+import com.researchspace.api.v1.controller.API_VERSION;
+import com.researchspace.model.User;
+import com.researchspace.model.oauth.UserConnection;
+import com.researchspace.model.oauth.UserConnectionId;
+import com.researchspace.service.UserConnectionManager;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.servlet.MvcResult;
+
+// @Ignore("We leave the test Ignored so we can potentially run it manually "
+//    + "by pasting the bearer token")
+public class FieldmarkRealConnectionMVCIT extends API_MVC_TestBase {
+
+  private static final String NOTEBOOK_ID = "1726126204618-rspace-igsn-demo";
+  private static final String ACCESS_TOKEN =
+      "eyJhbGciOiJSUzI1NiIsImtpZCI6ImZpZWxkbWFyayJ9.eyJfY291Y2hkYi5yb2xlcyI6WyIxNzI2MTI2MjA0NjE4LXJzcGFjZS1pZ3NuLWRlbW98fGFkbWluIl0sIm5hbWUiOiJSZXNlYXJjU3BhY2UgZGV2ZWxvcGVyIiwic3ViIjoiZGV2QHJlc2VhcmNoc3BhY2UuY29tIiwiaWF0IjoxNzI4MzkwODg1LCJpc3MiOiJEZW1vIEZpZWxkbWFyayBTZXJ2ZXIifQ.GxZfYvDYJiTTwr-rcVCjfarSOr21ezISK2YEptwbH0gZPVMC50Jf2PHHkMT6EQaILQCKHQIChHEjcsfhDoP_7yMZnVb6bwaUE2jSvDlRngKtWcym7dgcL7l5ZvXvMyTEGDw-ILLJMhqodViYbmJiipVc6JiS603pA2nynXF8FBE9m2P1Ml0ZR-ifuNRqfccY596EqKYz3qWXedPhmtDLeqlm4MEO5TZU-vlB3PsOqlw7ROx4mzDwRyEPZIUenyubf2wg5IGS9aN1W1N5YWWvrzAOQrcQvPHsjrO1SaG031MNK8XQrdq3HDykN84z4uYOPHWY-2_gV_UKCxDsRrlSOA";
```

Comment:
This access token should be removed and revoked since it's made it's way into this PR.